File: 392 - Is Subsequence/array.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:

        last = 0
        counter = 0

        for i in s:
            for j in range(last, len(t)):
                last += 1
                if i == t[j]:
                    counter += 1
                    break

        return counter == len(s)
        

File: 392 - Is Subsequence/array.go
func isSubsequence(s string, t string) bool {

	counter := 0
	last := 0

	for _, val := range s {
		for i := last; i < len(t); i++ {
			last++
			if string(val) == string(t[i]) {
				counter++
				break
			}
		}
	}

	return counter == len(s)
}